### PR TITLE
DEVEX-1765 app, applet builders for 16.04 v1 and 20.04

### PR DESCRIPTION
**Pull request by kpjensen**
Remote app and applet builders for 16.04 v1 and 20.04. Asset builder for 20.04.

### CODE REVIEW

File: tarball_app_builder_xenial_v1/dxapp.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "tarball_app_builder_xenial_v1",
+  "title": "Tarball App Builder (Xenial v1)",
+  "summary": "Builds an app from a tarball",
+  "categories": ["Debugging"],
+  "dxapi": "1.0.0",
+  "billTo": "org-dnanexus",
```
Comment (commandlinegirl) [inline tarball_app_builder_xenial_v1/dxapp.json:7]:
```suggestion
  "billTo": "org-dnanexus_apps",
```

Please bill these new apps to `org-dnanexus_apps`, an org dedicated to apps+public app project billing. Thanks!

File: Readme.md [hunk 2]
```diff
@@ -41,5 +41,6 @@ the build procedure are packed into the asset.
 * `create_asset_precise`: builds an asset targeting Ubuntu 12.04 (deprecated)
 * `create_asset_trusty`: builds an asset targeting Ubuntu 14.04
 * `create_asset_xenial`: builds an asset targeting Ubuntu 16.04
+* `create_asset_xenial_v1`: builds an asset targeting Ubuntu 16.04 version "1"
```
Comment (commandlinegirl) [inline Readme.md:44]: Please add a line for create_asset_focal

File: Readme.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ There are four apps here, each with different I/O specs:
   app in an Ubuntu 14.04 environment
 * `git_applet_builder`: takes a string containing a git repository path;
   returns an applet in an Ubuntu 14.04 environment
-* `tarball_app_builder[_trusty or _xenial]`: takes a reference to a tarball containing the code to
-  be compiled; builds an app [in an Ubuntu 14.04 or 16.04 environment]
-* `tarball_applet_builder[_trusty or _xenial]`: takes a reference to a tarball containing the code
-  to be compiled; returns an applet [in an Ubuntu 14.04 or 16.04 environment]
+* `tarball_app_builder[_trusty or _xenial or _xenial_v1]`: takes a reference to a tarball containing the code to
```
Comment (commandlinegirl) [inline Readme.md:20]: Add `focal`?

File: tarball_app_builder_xenial_v1/dxapp.json (hunk 1 above)
Comment (kpjensen) [reply tarball_app_builder_xenial_v1/dxapp.json]: Is `org-dnanexus` still used for developers?

Comment (commandlinegirl) [reply tarball_app_builder_xenial_v1/dxapp.json]: `org-dnanexus` should be removed from developers.

File: Readme.md (hunk 2 above)
Comment (commandlinegirl) [reply Readme.md:44]: If you could also update docs after this? https://github.com/dnanexus/developer-docs/blob/master/developer/apps/dependency-management/asset-build-process.md#dxassetjson-and-asset-metadata

- it says here only release "14.04" is accepted, oops :)

**Review by commandlinegirl — APPROVED** [review]
LGTM, thanks!

**Pull request merged**